src/components/ClaimENSPage/ClaimENSPage.css

```diff
@@ -33,13 +33,18 @@
   width: 240px;
 }
 
-.ClaimENSPage .dcl.field .overlay .ui.button.basic {
+.ClaimENSPage .dcl.section .dcl.field .overlay .ui.button.basic {
   font-size: 17px;
   color: var(--secondary-text) !important;
   cursor: default;
 }
 
+.ClaimENSPage .dcl.section.red .dcl.field .overlay .ui.button.basic {
+  color: red !important;
```

Comment:
Do we have a CSS variable to replace `red` somewhere else? Is `red` a DCL standard color? 

Is the `!important` really necessary?